status.go

```diff
@@ -174,11 +175,11 @@ func makeLocalConnectionStatusSlice(cm *connectionMaker) []LocalConnectionStatus
 					info = fmt.Sprintf("%-11v %v", "unencrypted", info)
 				}
 			}
-			slice = append(slice, LocalConnectionStatus{conn.remoteTCPAddress(), conn.isOutbound(), state, info})
+			slice = append(slice, LocalConnectionStatus{conn.remoteTCPAddress(), conn.isOutbound(), state, info, lc.OverlayConn.DisplayData()})
```

Comment:
I guess it's possible, but we also access `conn` 6 times with no locking - should we fix that too?